framework/components/ecl-link-blocks/ecl-link-blocks.twig

```diff
@@ -1,24 +1,38 @@
+{#
+  - "wrapper" (boolean): toggle display of a div wrapper around the component (default: false)
+  - "title" (string): optional title of the link block (default: '')
+  - "links" (array): [
+    "href" (string): target of the link
+    "label" (string): label of the link
+  ]
+  - "extra_classes" (string): extra CSS classes to be added
+  - "extra_attributes" (array): extra attributes classes (optional, format: [{ 'name': 'name_of_the_attribute', 'value': 'value_of_the_attribute'}])
+#}
+
+{% set wrapper = wrapper|default(false) %}
+{% set title = title|default('') %}
+
 {# Internal properties #}
 
-{% set _cssClass = 'ecl-link-block' %}
-{% set _extraAttributes = '' %}
+{% set _css_class = 'ecl-link-block' %}
+{% set _extra_attributes = '' %}
 
 {# Internal logic - Process properties #}
 
-{% if extraClass is defined %}
-  {% set _cssClass = _cssClass ~ ' ' ~ extraClass %}
+{% if extra_classes is defined %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
 {% endif %}
 
-{% if extraAttributes is defined %}
-  {% for attr in extraAttributes %}
-    {% set _extraAttributes = _extraAttributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+{% if extra_attributes is defined %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
   {% endfor %}
 {% endif %}
 
-{% if wrapper is defined and wrapper == true %}
+{% if wrapper == true %}
 
-<div class="{{ _cssClass }}" {{ _extraAttributes }}>
-  {% if title is defined %}
+<div class="{{ _css_class }}" {{ _extra_attributes }}>
+  {% if title != '' %}
```

Comment:
The `if title is not [empty](https://twig.symfony.com/doc/2.x/tests/empty.html)` might be handy.